formats/avro/src/main/avro/spec.avsc

```diff
@@ -0,0 +1,64 @@
+{
+  "namespace":"io.cloudevents",
```

Comment:
Using `io.cloudevents` fails the build due to generated code and module system, so let's use `io.cloudevents.avro` and update the imports accordingly: 
```suggestion
  "namespace":"io.cloudevents.avro",
```